components/motor/tmcstepper/stepper_motor_tmc.go

```diff
@@ -86,7 +91,6 @@ type Motor struct {
 	resource.Named
 	resource.AlwaysRebuild
 	resource.TriviallyCloseable
-	board       board.Board
```

Comment:
so now enLowPin is sufficient? board is no longer needed even if the user sets an enable pin?

Reply:
ah I see at the end of this file how board is being used for the enable pin, never mind!